app/src/main/java/de/danoeh/antennapod/fragment/preferences/UserInterfacePreferencesFragment.java

```diff
@@ -91,28 +102,89 @@ private void setupInterfaceScreen() {
         }
     }
 
-    private void showNotificationButtonsDialog() {
+    private void showCompatNotificationButtonsDialog() {
         final Context context = getActivity();
+
         final List<Integer> preferredButtons = UserPreferences.getCompactNotificationButtons();
         final String[] allButtonNames = context.getResources().getStringArray(
                 R.array.compact_notification_buttons_options);
+        final int[] buttonIDs = {0, 1, 2, 3};
```

Comment:
These magic numbers can lead to problems when other developers later change other parts of the code and then forget that the numbers are still here. Please use constants (either the ones from UserPreferences or an array in the resources forlder).